packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggle.js

```diff
@@ -32,6 +34,11 @@ DropdownToggle.propTypes = {
 };
 
 DropdownToggle.defaultProps = {
+  id:
```

Comment:
multiple dropdowns will have the same id this way, generate unique id in constructor instead

Reply:
Good catch.  Fixed this.